streaming/vision/convert/ade20k.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2022 MosaicML Composer authors
+
+"""ADE20K streaming dataset conversion scripts."""
+
+import os
+import random
+from argparse import ArgumentParser, Namespace
+from glob import glob
+from typing import Any, Dict, Iterable, List, Tuple
+
+from tqdm import tqdm
+
+from streaming.base import MDSWriter
+from streaming.vision.convert.base import get_list_arg
+
+
+def parse_args() -> Namespace:
+    """Parse command line arguments.
+
+    Args:
+        Namespace: Command line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--in',
```

Comment:
did lint really want half the flags 4-space indented and the other half first-line-space indented? oof!

Reply:
One unique way is to add `,` at the last parameter and yapf beautify it very well. Changed it in https://github.com/mosaicml/streaming/pull/5/commits/e1eaca7428674cb2941960d1ad7c39c7091c1b12.